### ISSUE TITLE
Issue with installing add-on, warns file appears corrupt

### ISSUE THREAD

**Issue by ddagson** [retorquere/zotero-better-bibtex#2125]
### TL;DR: 
I'm attempting to install Add on: _zotero-better-bibtex v6.5.1_ (latest version, when writing)
When attempting to install the add-on, I get the message: 
_"This add-on could not be installed because it appears to be corrupt."_

### Debug logs
Zotero Connector: D1736169539
Zotero: D1822248034

### My specs
System: _mac OS Catalina 10.15.7_
Browser: _Firefox 99.0.1 (64-bit)_
Standalone App: _Zotero 6.0.4_
Extension: _Zotero Connector 5.0.94_
(All of the above are the latest versions, when writing.)

### Detailed steps
I follow the installation instructions from https://retorque.re/zotero-better-bibtex/installation/.
- [x] I'm downloading the xpi-file from the latest release (v6.5.1), by right-clicking and selecting the option "save link as". File downloads successfully. 
- [x] In the main menu go to Tools > Add-ons
- [x] Select ‘Extensions’
- [x] Click on the gear in the top-right corner and choose ‘Install Add-on From File…’
- [ ] ?! Choose .xpi that you’ve just downloaded, click ‘Install’
**When I attempt this step, I get the following message:** 
"This add-on could not be installed because it appears to be corrupt."
- [ ] Restart Zotero

// David

**Comment by ddagson** [comment]
### Added Debug logs
Zotero Connector: D1736169539
Zotero: D1822248034

**Comment by ddagson** [comment]
Wow. Amazing. I was trying to install it in Firefox, when it explicitly said on the instruciton page to install it **in** the Zotero App. In other words, issue is resolved!
